ivadomed/evaluation.py

```diff
@@ -103,7 +103,7 @@ def evaluate(bids_df, path_output, target_suffix, eval_params):
             imed_inference.pred_to_png(painted_list,
                                        target_list,
                                        str(path_preds.joinpath(subj_acq)),
-                                       suffix="_painted")
+                                       suffix="_pred_painted.png")
```

Comment:
I don't know the ADS side of things, but in ivadomed I've seen a separation between [`suffix`](https://ivadomed.org/configuration_file.html#target-suffix) and [`extension`](https://ivadomed.org/configuration_file.html#extensions) as given in the configuration file. I understand that the `suffix` here serves another function, but perhaps for consistency sake we could add an `extension` parameter with the default value of `".png"` to the `pred_to_png` function?